### PR TITLE
BETA: Register venkat.t.is-a.dev

### PR DESCRIPTION
**Pull request by vnkt-t**
Added `venkat.t.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by 21Z — CHANGES_REQUESTED** [review]
404 Inaccessible website

**Comment by vnkt-t** [comment]
> 404 Inaccessible website

should work now

File: domains/venkat.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "1w.4",
+    "email": "popcrafts7@gmail.com"
+  },
+  "description": "this is for my personal website.",
+  "record": {
```
Comment (21Z) [inline domains/venkat.json:7]: site can't be reached

File: domains/venkat.t.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "vnkt-t",
+        "email": "venkat.k.terli@gmail.com"
+    },
+    "record": {
```
Comment (21Z) [inline domains/venkat.t.json:6]: site can't be reached

**Review by 21Z — CHANGES_REQUESTED** [review]

**Comment by 21Z** [comment]
Rename your website repo to `vnkt-t.github.io` from `a-website`

**Pull request closed without merging**